### PR TITLE
Add keyboard GTID entry support

### LINKED ISSUE #31: Allow manual GTID entry instead of tapping a BuzzCard

### PR DESCRIPTION
**Pull request by evan10s**
Testing:

1. Use the latest internal test build on Google Play
2. On the attendance screen there's a new Enter GTID manually button.
3. Go ham

Closes #31

### CODE REVIEW

File: base/src/main/java/org/robojackets/apiary/base/ui/nfc/BuzzCardPrompt.kt [hunk 1]
```diff
@@ -30,6 +39,8 @@ import java.nio.charset.StandardCharsets
  * As an alternative, you can use the hidePrompt parameter to hide the displayed UI elements from
  * this composable, making it hidden whilst also not allowing it to be disposed.
  */
+val gtidRegex = Regex("9[0-9]{8}")
```
Comment (kberzinch) [inline base/src/main/java/org/robojackets/apiary/base/ui/nfc/BuzzCardPrompt.kt:42]: I think we have pretty high confidence the second digit should be 0 for now

File: base/src/main/java/org/robojackets/apiary/base/ui/nfc/BuzzCardPrompt.kt [hunk 2]
```diff
@@ -66,7 +76,7 @@ fun BuzzCardPrompt(
                     buzzString = String(buzzData, StandardCharsets.UTF_8)
 
                     val buzzStringRegex = Regex("9[0-9]{8}=[0-9]{6}.*")
```
Comment (kberzinch) [inline base/src/main/java/org/robojackets/apiary/base/ui/nfc/BuzzCardPrompt.kt:78]: Maybe same here

File: base/src/main/java/org/robojackets/apiary/base/ui/nfc/BuzzCardPrompt.kt [hunk 3]
```diff
@@ -96,30 +106,100 @@ fun BuzzCardPrompt(
...
+                }
+            }) {
+                Text("Submit")
+            }
+        },
+        title = { Text(text = "Manual GTID entry", style = MaterialTheme.typography.h5) },
+        text = {
+            Column {
+                Text("Type the entire GTID, starting with 9")
+                OutlinedTextField(
```
Comment (kberzinch) [inline base/src/main/java/org/robojackets/apiary/base/ui/nfc/BuzzCardPrompt.kt:181]: Can you make the text field grab focus when the dialog appears? Right now I have to tap the text box to have the keyboard appear which adds a step.

**Review by kberzinch — CHANGES_REQUESTED** [review]

File: base/src/main/java/org/robojackets/apiary/base/ui/nfc/BuzzCardPrompt.kt (hunk 3 above)
Comment (evan10s) [reply base/src/main/java/org/robojackets/apiary/base/ui/nfc/BuzzCardPrompt.kt:181]: In theory, yes, but in practice I couldn't quite get it to work and don't think it's worth the trouble at the moment, considering this isn't intended to be a primary mode of use

**Review by kberzinch — APPROVED** [review]

**Pull request merged**